dashboards/redis/documentation.yaml

```diff
@@ -0,0 +1,62 @@
+exporter_type: sidecar
+app_name_short: Redis
+app_name: {{app_name_short}}
+app_site_name: Redis
+app_site_url: https://www.redis.com/
+exporter_name: Redis
+exporter_pkg_name: redis_exporter
+exporter_repo_url: https://github.com/oliver006/redis_exporter
+dashboard_available: true
+minimum_exporter_version: v1.43.1
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+config_mods: |
+  apiVersion: apps/v1
+  kind: StatefulSet
+  metadata:
+    name: redis
+  spec:
+    serviceName: redis
+    selector:
+      matchLabels:
+  +     app.kubernetes.io/name: redis
+    template:
+      metadata:
+        labels:
+  +       app.kubernetes.io/name: redis
+      spec:
+        containers:
+        - name: redis
+          image: "redis:6.2"
+          ports:
+          - containerPort: 6379
+  +     - name: redis-exporter
+  +       image: oliver006/redis_exporter:v1.43.1
+  +       args: [--include-system-metrics]
+  +       resources:
+  +         requests:
+  +           cpu: 100m
+  +           memory: 100Mi
+  +       ports:
+  +       - containerPort: 9121
+  +         name: prometheus
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: redis
+    labels:
+      app.kubernetes.io/name: redis
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+    - port: prometheus
```

Comment:
In our original iteration, we used a numbered port, but agree that going forward named ports are preferable when possible.

Reply:
Thanks for the review. On our end, we are trying to be as consistent as possible with things like this.